docs/root/version_history/current.rst

```diff
@@ -1,5 +1,5 @@
-1.16.0 (Pending)
-================
+1.16.0 (October 8, 2020)
+========================
```

Comment:
is the `//` intended in
 `build: added visibility rules for upstream. If these cause visibility related breakage, see notes in //BUILD.`?